app/file.go

```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/mattermost/platform/model"
 	"github.com/mattermost/platform/utils"
 	s3 "github.com/minio/minio-go"
+	credentials "github.com/minio/minio-go/pkg/credentials"
```

Comment:
Yes, this requires minio-go 3.0 (I updated the glide.yml, but was under the impression that I shouldn't actually update glide.lock or the vendors directory.